pkg/controller/daemon/daemoncontroller_test.go

```diff
@@ -1219,6 +1219,19 @@ func TestUpdateNode(t *testing.T) {
 			ds:            newDaemonSet("ds"),
 			shouldEnqueue: true,
 		},
+		{
```

Comment:
Added a test when some conditions are false; in this PR, user's case is about out of disk but our logic does not tight to it.

I'll open another PR for resources changed, not solution for that case now.